pipeline/pipelines/core_generation_pipeline.py

```diff
@@ -17,20 +17,21 @@
    language."""
 
 from pipeline.pipelines import code_generation_pipeline
+from pipeline.pipelines import grpc_generation_pipeline
 from pipeline.tasks import protoc_tasks
+from pipeline.tasks import publish_tasks
 
 
-class JavaCoreProtoPipeline(
-        code_generation_pipeline.CodeGenerationPipelineBase):
+class JavaCoreProtoPipeline(grpc_generation_pipeline.GrpcClientPipeline):
```

Comment:
Add a comment explaining why this inherits from the grpc pipeline. Also maybe consider inheriting directly from JavaGrpcClientPipeline (and updating the packman_flags)

I can see arguments for no inheritance from grpc, inheriting from the base grpc pipeline (as you have done), and inheriting from the java grpc pipeline, so in any case I think some explanation would be good.